internal/integration/api/apply-config.go

```diff
@@ -239,22 +239,18 @@ func (suite *ApplyConfigSuite) TestApplyConfigRotateEncryptionSecrets() {
 
 	encryption := machineConfig.MachineConfig.MachineSystemDiskEncryption
 
-	if encryption == nil {
+	if encryption == nil { //nolint:staticcheck
 		suite.T().Skip("skipped in not encrypted mode")
 	}
 
-	cfg := encryption.EphemeralPartition
```

Comment:
this looks weird, could it be that your `make lint` got wrong cache and now reporting wrong issues?

Reply:
oh yes, must be that, how can I fix it?

Reply:
sometimes just `docket stop` for the buildx container is enough

sometimes `docker rm -f` for the buildx container helps (as it kills the cache)

you can also try `docker buildx prune`

Reply:
yep, the last command solved it. reverted.